.github/ISSUE_TEMPLATE/Feature_request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨
+
+Half of all issues filed here are duplicates, answered in the FAQ, or not appropriate for the bug tracker. Please read the FAQ first, especially the "Common Feature Requests" section.
+
+-->
+
+## Search Terms
+
+<!-- List of keywords you searched for before creating this issue. Write them down here so that others can find this suggestion more easily -->
+
+## Suggestion
+
+<!-- A summary of what you'd like to see added or changed -->
+
+## Use Cases
+
+<!--
+What do you want to use this for?
+What shortcomings exist with current approaches?
+-->
+
+## Examples
+
+<!-- Show how this would be used and what the behavior would be -->
+
+## Checklist
+
+I attest that all of the following are true:
+ [ ] This wouldn't change the behavior of existing JavaScript code
+ [ ] This could be implemented without emitting different JS based on the types of the expressions
+ [ ] This isn't a runtime feature that should be handled by TC39 instead

```

Comment:
"should be handled by TC39" is a bit subjective. i would make `it does this introduce any new syntax`